### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_Summary_ScreenReader:The feedback message after activating copy button is not announced.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9322]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yWCAG2.1;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#JAWS;#NVDA;#MAS4.1.3;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader:JAWS,NVDA
MAS References:MAS4.1.3

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Select Configuration profile --> Update Azure settings --> Update Cluster settings --> Update Service settings & hit Next.
8. Launch screen reader as NVDA and navigate to the copy button.

**Actual:**
On navigating using tab to the copy button press enter there is no feedback message about the same action.

Issue observed with JAWS

**Expected:**
The status message should be announced and there should be a message visually seen.

**User Impact:**
Screen reader users will not know if the action has been performed.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EW0NnOOn65BDhlFgcTrOZIgBMF4hYbAjvIONY-bOFf7oAA?e=9piv2e

**Comment by alanrenmsft** [comment]
how do we let user know the action is finished.

**Comment by alanrenmsft** [comment]
I will ask the accessibility team for guideline

**Comment by Charles-Gagnon** [comment]
Discussed this in office hours with v-tayazd - for actions which execute immediately such as this we shouldn't be announcing that an action is occurring as that tends to be too verbose.

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_Win32;#A11yMAS;

**Comment by BhanuPrakashGit** [comment]
As "SQL Server Big Data Cluster" feature has been removed.
**GithubTags:**#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
